backend/src/main/java/com/rcbsukcesja/hack2react/model/entity/User.java

```diff
@@ -67,15 +68,11 @@ public class User {
     @JoinColumn(name = "organization_id", referencedColumnName = "id")
     private Organization organization;
 
-    @ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE,
-            CascadeType.REFRESH, CascadeType.DETACH})
-    @JoinTable(
-            name = "conversation_users",
-            schema = "wavvy",
-            joinColumns = {@JoinColumn(name = "user_id")},
-            inverseJoinColumns = {@JoinColumn(name = "conversation_id")}
-    )
-    private List<Conversation> conversations;
+    @OneToMany(mappedBy = "receiver",cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    private List<Message> receivedMessages;
+
+    @OneToMany(mappedBy = "sender",cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    private List<Message> sentMessages;
```

Comment:
Chyba jednak nie chcemy żeby wysłane wiadomości ginęły jak usuniemy użytkownika. Wtedy:
```suggestion
    @OneToMany(mappedBy = "sender",cascade = CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH, CascadeType.DETACH, fetch = FetchType.LAZY)
    private List<Message> sentMessages;
```